staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -358,6 +360,12 @@ type HotplugVolumeStatus struct {
 	AttachPodUID types.UID `json:"attachPodUID,omitempty"`
 }
 
+// ContainerDiskInfo shows info about the containerdisk
+type ContainerDiskInfo struct {
+	// Checksum is the checksum of the rootdisk inside the containerdisk
```

Comment:
This mentions only rootdisk. What about the case with ext. kernel and initrd? I guess those should also be checksummed.

Reply:
+1

Reply:
they are checksummed as well since they're plain containerdisks, I'll fix the comment.

Reply:
fixed, now kernel and initrd are checksummed as well.
Thanks for noticing!